modules/vantagecloud-lake/pages/vantagecloud-lake-demo-jupyter-azure.adoc

```diff
@@ -0,0 +1,183 @@
+= Run Teradata Jupyter Notebook Demos for VantageCloud Lake in Azure
+:experimental:
+:page-author: Daniel Herrera
+:page-email: daniel.herrera2@teradata.com
+:page-revdate: January 26th, 2024
+:description: Run Teradata Jupyter Notebook Demos for VantageCloud Lake in Azure
+:keywords: data warehouses, compute storage separation, teradata, vantage, cloud data platform, business intelligence, enterprise analytics, jupyter, teradatasql, ipython-sql, cloud computing, machine learning, sagemaker, vantagecloud, vantagecloud lake, lake
+:dir: vantagecloud-lake-demo-jupyter-azure
+:auxdir: vantagecloud-lake-demo-jupyter-sagemaker
+
+== Overview
+This quickstart details the process for running the https://github.com/Teradata/lake-demos[Teradata Jupyter Notebook Demos for Vantage Cloud Lake], on Microsoft Azure.
+
+== Prerequisites
+* Access to a Microsoft Azure account
+* https://quickstarts.teradata.com/getting-started-with-vantagecloud-lake.html[Access to a VantageCloud Lake environment]
+
+== Microsoft Azure setup
+In this section we will cover in detail each of the steps below:
+
+* Create a Microsoft Azure Web App based on Teradata Jupyter Lab extensions Docker image
+* Configure Jupyter Lab extensions Azure Web App
+* Load Vantagecloud Lake demos to Jupyter Lab extensions Azure Web App
+* Find the IP of the Jupyter Lab extensions Azure Web App
+
+=== Create a Microsoft Azure Web App based on Teradata Jupyter Lab extensions Docker image
+* Login to Microsoft Azure and click on APP Services 
+
+image::{dir}/azure-console-0.PNG[Azure console,align="center" width=75%]
+
+* On App Services click Web App
+
+image::{dir}/azure-app-service-1.PNG[Create Azure web app,align="center" width=75%]
+
+* On the Basics tab:
+** Select the appropiate resource group
+** Enter a name for your web app
+** On Publish select Docker Container 
+** Select Linux as operating system
+** Select the appropiate region according to your needs
+** Select the appropiate App Service plan, if you don't have any a new one will be created with default configurations
+** For purposes of the VantageCloud Lake demo redundancy is not needed
+
+image::{dir}/azure-app-service-2.PNG[Create Azure web app Basics,align="center" width=75%]
+
+* On the Docker tab:
+** Select Single Container, the default option
+** On Image Source select Docker Hub
+** On Access Type select Public
+** On Image and tag type `teradata/jupyterlab-extensions:latest`
+** A startup command is not needed for this App Service
+
+image::{dir}/azure-app-service-3.PNG[Create Azure web app Docker,align="center" width=75%]
```

Comment:
Again, perhaps this should be in the next bullet point?

Reply:
Not really, same comment as before.